### ISSUE TITLE
Populate with buffers

### ISSUE THREAD

**Issue by GuiMarthe** [ThePrimeagen/harpoon#141]
Is there a way to quickly mark all the current buffers?

The idea would be instead of manually marking each open buffer, just add the currently open buffers.
If this is a good idea, I could try and implement it.

**Comment by joshmedeski** [comment]
I don't think this is too complicated. We just have to loop through the buffer list and add each file to harpoon. I can experiment with it in the next month and report back.
